src/Classes/ServiceAPI/MyRadio_List.php

```diff
@@ -67,6 +67,13 @@ class MyRadio_List extends ServiceAPI
     private $members = [];
 
     /**
+     * This is the set of members have opted out of this 'optin' list.
+     *
+     * @var int[]
+     */
+    private $nonMembers = [];
```

Comment:
For opt-in lists, the table represents members who are opted-in. For autogenerated etc lists, it represents members who are opted *out*.